### PR TITLE
Update quoting.md with Snowflake example

### PR DESCRIPTION
**Pull request by tjirab**
## Description & motivation
Proposing to add an example how to cope with Snowflake reserved keywords as table names. This is relevant when the table is generated by a 3rd party tool (e.g. Stitch, Fivetran), you cannot change its name, and you need to add it as source. In the proposed change there's an example how to configure the .yml file to allow users to refer to it in `source()` in downstream models.

## To-do before merge
NA

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [x] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

## Checklist
NA

### CODE REVIEW

**Comment by runleonarun** [comment]
@dbeatty10 Can you provide a technical review for this PR? Thank you! 🙇🏻

**Review by dbeatty10 — COMMENTED** [review]
This is a helpful call-out as it relates to `quoting` in Snowflake.

Rather than include a YAML example in this webpage, I'd suggest a single sentence similar to the following instead:

> If a Snowflake source table uses a quoted database, schema, or table identifier, it can be configured per the instructions [here](https://docs.getdbt.com/reference/resource-properties/quoting).

File: website/docs/reference/project-configs/quoting.md [hunk 1]
```diff
@@ -112,6 +112,19 @@ create table analytics.dbt_alice.dim_customers
 ### Snowflake
 Set all quoting configs to `False`. This means that you cannot use reserved words as identifiers, however it's usually a good idea to avoid these reserved words anyway.
 
+If you cannot change the source table name, quote the specific table name in the source.yml file:
```
Comment (mirnawong1) [inline website/docs/reference/project-configs/quoting.md]:
```suggestion
If a Snowflake source table uses a quoted database, schema, or table identifier, you can configure it in the source.yml file. [Refer to configuring quoting for more info](/reference/resource-properties/quoting).
```

File: website/docs/reference/project-configs/quoting.md [hunk 2]
```diff
@@ -112,6 +112,19 @@ create table analytics.dbt_alice.dim_customers
...
+If a Snowflake source table uses a quoted database, schema, or table identifier, you can configure it in the source.yml file. [Refer to configuring quoting for more info](/reference/resource-properties/quoting).
+
+```yml
+sources:
+  - name: <your_schema>
+    tables:
+      - name: group
+        identifier: GROUP
+        quoting:
+          identifier: true
```
Comment (mirnawong1) [inline website/docs/reference/project-configs/quoting.md]:
```suggestion
```

File: website/docs/reference/project-configs/quoting.md [hunk 3]
```diff
@@ -112,6 +112,11 @@ create table analytics.dbt_alice.dim_customers
 ### Snowflake
 Set all quoting configs to `False`. This means that you cannot use reserved words as identifiers, however it's usually a good idea to avoid these reserved words anyway.
 
+If a Snowflake source table uses a quoted database, schema, or table identifier, you can configure it in the source.yml file. [Refer to configuring quoting for more info](/reference/resource-properties/quoting).
+
+
+```
```
Comment (mirnawong1) [inline website/docs/reference/project-configs/quoting.md]:
```suggestion
```

**Review by mirnawong1 — COMMENTED** [review]
hey @dbeatty10 - i've added your excellent suggestion. let me know what you think and will enable auto merge .

**Review by dbeatty10 — APPROVED** [review]

**Pull request merged**